packages/webcert/src/feature/certificate/Buttons/SignAndSendButton.tsx

```diff
@@ -22,7 +21,7 @@ const SignAndSendButton: React.FC<Props> = ({ name, description, enabled, functi
       tooltip={description}
       buttonStyle="primary"
       text={name}
-      startIcon={<FontAwesomeIcon size="lg" icon={faFileSignature} />}
+      startIcon={<img src={edit} alt="Logo Signera intyget" />}
```

Comment:
Got tired of adding the same message, you get the picture, right? :)

Reply:
Yes, it's better if you add it to one comment especially when it's a comment that relates to opinion rather than implementation